src/styles/calendar.scss

```diff
@@ -0,0 +1,51 @@
+$weekdays: mon 0, tue 1, wed 2, thu 3, fri 4, sat 5, sun 6;
+
+.calendar {
```

Comment:
If you add to this class `margin: 0 auto;` then the calendar is aligned horizontally in the center.